openshift/settings.sh

```diff
@@ -1,10 +1,9 @@
 export PROJECT_NAMESPACE="7cba16"
 export GIT_URI="https://github.com/bcgov/von-bc-registries-agent.git"
 export GIT_REF="master"
+export SKIP_PIPELINE_PROCESSING=1
```

Comment:
Do we really want this committed?

Reply:
not sure how that got there... it might have been from when I merged knp branch into here

Reply:
my git lens tells me that this line was added by @WadeBarnes a month ago